style.css

```diff
@@ -639,22 +672,18 @@ input:focus:invalid {
     line-height: 2.5rem;
   }
 
-  #social-media {
-    position: relative;
-    top: -18.5rem;
-    left: 3rem;
-  }
-
-  #social-media > ul {
-    height: 12.5rem;
+ 
+  #social-media > ul{
+     position: absolute;
+    margin-top: -26rem;
+    margin-left: -5rem;
```

Comment:
- [ ] Kindly fix the positioning of the social media icons as in larger screen sizes, it overlaps the content: 
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/91486791/60b78042-5852-4773-82c5-10b1de146075)